Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12
+
+# update and upgrade libraries
+RUN apt-get -qq update \
+    && apt-get -qq upgrade \
+	&& rm -rf /var/lib/apt/lists/ \
+	&& apt-get -qq clean
+
+# create mitodl user
+RUN adduser --disabled-password --gecos "" mitodl
```

Comment:
It looks like there is already a `node` user who took the `1000` uid. I guess we can just reuse that user instead of adding mitodl here.